cmd/syncthing/decrypt/decrypt.go

```diff
@@ -88,7 +89,7 @@ func (c *CLI) walk() error {
 		if !info.IsRegular() {
 			return nil
 		}
-		if fs.IsInternal(path) {
+		if fs.IsInternal(path) && !strings.HasPrefix(path, ".stversions") {
```

Comment:
The string should be defined as a constant once instead of repeating it everywhere. Please check whether there is already some central place where it is defined.

It might even depend on the versioning configuration? I'm not sure whether the path can be configured for simple, trashcan and staggered versioning, but at least for external we cannot know a constant path. That should be respected in the code or at least a comment regarding the limitation.

Reply:
No public shared space for constants, left one copy in added code

Reply:
We could perhaps instead look at whether the path contains `.syncthing-enc` or not